### ISSUE TITLE
Debug code?

### ISSUE THREAD

**Issue by davidjosephhayes** [teamsinger/uk.teamsinger.civicrm.mailgun#14]
Is the echo on line 278 here: https://github.com/teamsinger/uk.teamsinger.civicrm.mailgun/blob/master/CRM/Core/IDS.php supposed to still be there?

**Comment by tomsinger** [comment]
No, you're right in that it was some debug code. To be honest the whole file is only added as a workaround to allow extra paths to skip IDS checks.

I'm planning on putting a pull request together for this but needed to launch this before I had a chance to hence the very hacky workaround of commiting this patched file.

**Comment by davidjosephhayes** [comment]
I like your method for allowing certain extensions to skip IDS checks. I think it should be in the core as well.

The setup currently is ok IMO though. CiviCRM uses the IDS.php you used if you upload it to the extension directory without overwriting the core IDS.php. It's not optimal because if another extension tries to customize IDS.php, its uncertain which file will get used.
